modules/javascript-udfs/partials/nav.adoc

```diff
@@ -1,4 +1,4 @@
-* xref:javascript-udfs:javascript-functions-with-couchbase.adoc[]
- ** xref:javascript-udfs:calling-javascript-from-n1ql.adoc[]
- ** xref:javascript-udfs:calling-n1ql-from-javascript.adoc[]
- ** xref:javascript-udfs:handling-errors-javascript-udf.adoc[]
+*** xref:javascript-udfs:javascript-functions-with-couchbase.adoc[]
+ **** xref:javascript-udfs:calling-javascript-from-n1ql.adoc[]
+ **** xref:javascript-udfs:calling-n1ql-from-javascript.adoc[]
+ **** xref:javascript-udfs:handling-errors-javascript-udf.adoc[]
```

Comment:
I would delete the `javascript-udfs` nav partial and incorporate its content into the `n1ql` nav partial, same as you have with the `learn` nav partial. Makes things a lot simpler.